push_dockerhub.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+echo $DOCKER_PASSWORD | docker login --username $DOCKER_USERNAME --password-stdin
+docker tag $timestamp_holder $DOCKER_REPO
+docker push $DOCKER_REPO
+
+#docker rmi $latest_holder
+#docker rmi $timestamp_holder
```

Comment:
I suggest removing these 2 rows, and handle the (conditional) removal of the images on the `runner.sh` level. That way it will be consistent for all 3 `push_*.sh` files.

Reply:
Done.